app/assets/stylesheets/base/_fonts.scss

```diff
@@ -5,22 +5,22 @@
 
 @font-face {
   font-family: 'bpicons';
-  src:url('bpicons.eot');
-  src:url('bpicons.eot?#iefix') format('embedded-opentype'),
-    url('bpicons.woff') format('woff'),
-    url('bpicons.ttf') format('truetype'),
-    url('bpicons.svg#bpicons') format('svg');
+  src:font-url('bpicons.eot');
```

Comment:
Colon after property should be followed by one space<br>Prefer double-quoted strings